doc/sphinx/proof-engine/ltac2.rst

```diff
@@ -860,6 +860,8 @@ in a less hard-wired way.
       dependencies in the variables :n:`@ident__i` and returns a functional term
       of the form :n:`fun @ident__1 … @ident__n => @term`.
 
+   .. _match_term_context_2:
+
```

Comment:
Why is this needed?  I didn't find any link referring to it.

Reply:
I think this is the rest of a change I made to avoid a compilation warning (treated as a fatal error). It is apparently useless now, so I’m removing this.